AirCasting/UIState/UIStateHandler.swift

```diff
@@ -0,0 +1,35 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import Foundation
+import Resolver
+
+protocol UIStateHandler {
+    func cardToggle(sessionUUID: SessionUUID)
+    func streamChange(sessionUUID: SessionUUID, newStream: String)
+}
+
+class UIStateHandlerDefault: UIStateHandler {
+    @Injected private var UIStorage: UIStorage
+    
+    func cardToggle(sessionUUID: SessionUUID) {
+        UIStorage.accessStorage { storage in
+            do {
+                try storage.cardStateToggle(for: sessionUUID)
+            } catch {
+                Log.info("(UIState) Error toggling card state \(error)")
```

Comment:
`(UIState)` is not needed since logger will log automatically which file, function and line the log originates from (when my PR is merged). Also this should be warning or error log, not info